osf/models/notable_domain.py

```diff
@@ -40,6 +42,18 @@ def __repr__(self):
     def __str__(self):
         return repr(self)
 
+    @staticmethod
+    def has_spam_domain(content):
```

Comment:
As stated elsewhere, I don't think we need this, but if we were to keep it, why pass the content instead of just the discovered domains?